engine/src/main/java/org/terasology/rendering/nui/widgets/UIText.java

```diff
@@ -68,27 +68,27 @@
     private TextureRegion cursorTexture;
 
     @LayoutConfig
-    private Binding<String> text = new DefaultBinding<>("");
+    protected Binding<String> text = new DefaultBinding<>("");
 
     @LayoutConfig
-    private boolean multiline;
+    protected boolean multiline;
 
     @LayoutConfig
     private boolean readOnly;
```

Comment:
Should I redefine the methods as well? They do have the exact same functionality as in the parent class and will lead to a lot of code repetition.